addons/l10n_eg_edi_eta/models/account_move.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from datetime import datetime
+
+from odoo.exceptions import ValidationError, UserError
+
+from odoo import models, fields, api, _
+import logging
+
+import json
```

Comment:
import order is wrong, worth double-checking in other files too:
https://www.odoo.com/documentation/15.0/developer/misc/other/guidelines.html#imports